src/services/arguments-manager/arguments-manager.service.ts

```diff
@@ -1,23 +1,28 @@
-import _ from "lodash"
-import { Message } from "discord.js"
+import _ from "lodash";
+import { Message } from "discord.js";
+import { CommandsEnum } from "../../enums/commands.enum";
 
 export class ArgumentsManagerService {
-  private static _instance: ArgumentsManagerService
+  private static _instance: ArgumentsManagerService;
 
   public static getInstance(): ArgumentsManagerService {
     if (_.isNil(ArgumentsManagerService._instance)) {
-      ArgumentsManagerService._instance = new ArgumentsManagerService()
+      ArgumentsManagerService._instance = new ArgumentsManagerService();
     }
-    return ArgumentsManagerService._instance
+    return ArgumentsManagerService._instance;
   }
 
-  public getArguments(message: Message, botPrefix: string) {
-    return message.content.slice(botPrefix.length).split(/ +/)
+  public getArguments(message: Message, botPrefix: string): string[] {
+    return message.content.slice(botPrefix.length).split(/ +/);
   }
 
-  public extractCommand(entries: string[]) {
-    const usableBotCommand = entries.shift()
-    if (usableBotCommand) return usableBotCommand.toLowerCase()
-    return
+  public extractCommand(entries: string[]): string {
```

Comment:
To enhance the types, you should return `string | CommandsEnum.NONE`

Reply:
Ok, I see. I wil add '| CommandsEnum.NONE'.